src/evt_drivers/evt_pins.c

```diff
@@ -1,20 +1,31 @@
 #include "evt_pins.h"
 
 #include "chip.h"
-void Evt_GPIO_Init(void) {
-    // LEDs
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED0);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED1);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED2);
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_LED2, IOCON_FUNC1);	
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED0, true);
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED1, true);
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED2, true);
 
+// CTR Pin
+#define EVT_CTR_SWTCH 1, 4 // pin 40 on lpc24, used as input 
+#define EVT_IOCON_CTR_SWTCH IOCON_PIO1_4
+#define EVT_IOCON_CTR_SWTCH_FUNC IOCON_FUNC0
+
+// Headroom Pin
+#define EVT_HEADROOM 3, 2
+#define EVT_IOCON_HEADROOM IOCON_PIO3_2
+#define EVT_IOCON_HEADROOM_FUNC IOCON_FUNC1
```

Comment:
Probably just a bug. We probably never even used IOCON_PinMuxSet for headroom so that's why it didn't show up. Nice catch!